src/test/java/de/qabel/desktop/ui/accounting/item/AccountingItemControllerTest.java

```diff
@@ -37,15 +35,4 @@ private AccountingItemController getController(Identity identity) {
         params.put("identity", identity);
         return params;
     }
-
-    @Test
-    public void savesAlias() throws Exception {
-        Identity i = new Identity("alias", null, new QblECKeyPair());
-        AccountingItemController controller = getController(i);
-        controller.setAlias("new alias");
-        assertEquals("new alias", controller.alias.getText());
-        Identities results = identityRepository.findAll();
-        Identity identity = results.getByKeyIdentifier(i.getKeyIdentifier());
-        assertEquals("new alias", identity.getAlias());
-    }
```

Comment:
This test is still of value. It verifies that the `identityRepository.save` in the IdentityContextMenuController line 126 is really done. Without this test I could remove that line of production code and nobody would notice because no other test covers it. It just has to be moved to be an IdentityContextMenuControllerTest because the corresponding code was moved there from the AccountingItemController.